crates/bevy_ecs/src/storage/resource.rs

```diff
@@ -99,6 +100,20 @@ impl<const SEND: bool> ResourceData<SEND> {
         })
     }
 
+    pub(crate) fn get_mut(
```

Comment:
Feels like this could be used elsewhere to simplify code. Might require a conversion to `ResMut<T>` though.